listen.js

```diff
@@ -0,0 +1,7 @@
+const app = require('./app')
```

Comment:
we don't need any of this until ticket 14 and we start thinking about hosting, supertest has it's own built in listener